cmd/dep/ensure.go

```diff
@@ -304,27 +304,23 @@ func getProjectConstraint(arg string, sm gps.SourceManager) (gps.ProjectConstrai
 }
 
 func checkErrors(m map[string]pkgtree.PackageOrErr) error {
```

Comment:
Can you also add a simple table test for this?

We lost `ensure_test.go` recently. You would have to create that file again. 🙂